impl/src/lib.rs

```diff
@@ -12,12 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-pub mod bazel;
 pub mod context;
-pub mod license;
+pub mod error;
 pub mod metadata;
-pub mod planning;
-pub mod rendering;
+pub mod planning {
```

Comment:
This seems weird.

Why not src/planning/mod.rs? Would also allow you to ditch the weird "planning::planning"